### ISSUE TITLE
Keyboard does not work on custom submenu entry

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#110765]
Issue Type: <b>Bug</b>

Repro:

1. Shift+F10 to open context menu
2. Use arrow keys to select Peek submenu entry
3. Press right arrow

:bug: => nothing happens

VS Code version: Code - Insiders 1.52.0-insider (8f9aeb83da26902e21ca41f8d17d204d2ac216cf, 2020-11-16T11:24:42.172Z)
OS version: Windows_NT x64 10.0.19042
